src/serialize.rs

```diff
@@ -271,6 +281,11 @@ fn test_serialized_length_from_bytes() {
     );
 }
 
+#[cfg(test)]
+fn encode_size(f: &mut dyn Write, size: u64) -> io::Result<()> {
```

Comment:
Some tests use this now-deprecated function, so we reimplement it in terms of a non-deprecated replacement.